person.rb

```diff
@@ -21,6 +20,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
+
```

Comment:
- [ ] Here again, let's ensure the _person_ itself is sent as a parameter to create the new _rental_, thereby achieving the _'has-many'_ association. We can achieve this by modifying our code to look like the following example below:

```ruby
class Person
  attr_accessor :name, :age, :rentals
  .
  .
  def initialize(age, name = "Unknown", parent_permission = true)
    .
    .
   @rentals = []
  end
  .
  .

  def add_rental(book, date)
    Rental.new(date, book, self)
  end

  .
  .
end
 ```

- [ ] In addition, let's mark `rental` with `attr_acessor` and initialize @rentals in a constructor, like the code above.